pylink/jlink.py

```diff
@@ -545,6 +545,12 @@ def open(self, serial_no=None, ip_addr=None):
 
         if ip_addr is not None:
             addr, port = ip_addr.split(':')
```

Comment:
We should actually make this `addr, port = ip_addr.rsplit(':', 1)` to address the earlier problem that you raised in the issue #36